test/cypress/integration/playing_vs_computer/win_easy_opponent.spec.js

```diff
@@ -0,0 +1,21 @@
+import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
+addMatchImageSnapshotCommand();
+
+describe('Win easy opponent', function(){
+  it('Opens the game', function() {
+    cy.visit('/')
+    cy.matchImageSnapshot('Expected Start Menu');
+  })
+
+  it('Starts a game round against an easy bot', function() {
+    cy.wait(10)
```

Comment:
Any particular need to wait?

Reply:
Some issue with loading font-awesome symbols